recipes/lib-pod5/build.sh

```diff
@@ -0,0 +1,3 @@
+VERSION="0.2.4"
+PYTHON_VERSION=`python -c 'import sys; version=sys.version_info[:3]; print("{0}{1}".format(*version))'`
+pip install https://files.pythonhosted.org/packages/cp$PYTHON_VERSION/l/lib_pod5/lib_pod5-$VERSION-cp$PYTHON_VERSION-cp$PYTHON_VERSION-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
```

Comment:
Are you sure this is really necessary? Why is it not possible to use a normal `pip install` command?

Reply:
I wanted to make sure the correct python version is taken for the package using the wheel file from pypi. Using a simple pip install did not work for me, as it returned the error `ModuleNotFoundError: No module named 'lib_pod5'`. Any suggestions?

Reply:
I removed the file.
Now the package is installed via the command:
`{{ PYTHON }} -m pip install https://files.pythonhosted.org/packages/cp311/l/lib_pod5/lib_pod5-{{ version }}-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl --no-deps --ignore-installed --no-cache-dir -vvv`